src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -325,10 +333,19 @@ namespace NServiceBus
     {
         public static NServiceBus.ConventionsBuilder DefiningExpressMessagesAs(this NServiceBus.ConventionsBuilder builder, System.Func<System.Type, bool> definesExpressMessageType) { }
     }
+    [System.ObsoleteAttribute("Message property encryption is released as a dedicated \'NServiceBus.Encryption.Me" +
+        "ssageProperty\' package. Will be treated as an error from version 7.0.0. Will be " +
+        "removed in version 8.0.0.", false)]
     public class EncryptedValue
     {
         public EncryptedValue() { }
+        [System.ObsoleteAttribute("Message property encryption is released as a dedicated \'NServiceBus.Encryption.Me" +
```

Comment:
@andreasohlund not sure, it doesn't seem the classes obsolete message when working with references in case the used property isn't obsoleted. Not sure why it behaves that way, but this is how it looks on my machine with obsolete:

![image](https://cloud.githubusercontent.com/assets/3524870/22883264/b2105c94-f1ef-11e6-8228-1e08c0b9c1cb.png)

When I remove the obsolete on `Value`:

![image](https://cloud.githubusercontent.com/assets/3524870/22883291/cc347042-f1ef-11e6-9e88-8aeb8dada6d7.png)

of course you can argue that the warning on the property type would be good enough to point out the obsoletion. No strong feelings tbh, but It doesn't seem to hurt?